lib/net/imap/response_parser.rb

```diff
@@ -1119,6 +1125,20 @@ def resp_text_code
         when /\A(?:UIDVALIDITY|UIDNEXT|UNSEEN)\z/n
           match(T_SPACE)
           result = ResponseCode.new(name, number)
+        when /\A(?:APPENDUID)\z/n
+          match(T_SPACE)
+          uidvalidity = number
+          match(T_SPACE)
+          append_uid = number
+          result = ResponseCode.new(name, [uidvalidity, append_uid])
+        when /\A(?:COPYUID)\z/n
+          match(T_SPACE)
+          uidvalidity = number
+          match(T_SPACE)
+          from_uid = set
```

Comment:
Can we rename this (see my comment on `def set` below)?

```suggestion
          from_uid = uid_set
```